### PR TITLE
Added example for echo

### PR DESCRIPTION
**Pull request by hiyali**
Thanks a lot for great works.

### CODE REVIEW

File: examples/echo/main.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	log "github.com/hiyali/logli"
```
Comment (novln) [inline examples/echo/main.go]: Could you remove this import for the example and use `log` ?

**Comment by novln** [comment]
Thank you for this pull request.

**Comment by hiyali** [comment]
Cleaned up.

**Comment by hiyali** [comment]
Cleaned up.

**Comment by novln** [comment]
Thank you 🙂

**Pull request merged**